example/lib/main.dart

```diff
@@ -94,6 +96,7 @@ class Count extends StatelessWidget {
 
         /// Calls `context.watch` to make [Count] rebuild when [Counter] changes.
         '${context.watch<Counter>().count}',
+        key: const Key('counterState'),
```

Comment:
keys should be the first parameter passed to widgets

Reply:
But Text widget needs string first
```Text(String data, {Key key, TextStyle style, StrutStyle strutStyle, TextAlign textAlign, TextDirection textDirection, Locale locale, bool softWrap, TextOverflow overflow, double textScaleFactor, int maxLines, String semanticsLabel, TextWidthBasis textWidthBasis, TextHeightBehavior textHeightBehavior})```

Reply:
Oh yeah forget that I didn't see the full code